helm-chart/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+name: latest
+description: A generated Helm Chart for latest from Skippbox Kompose
+version: 0.0.1
+apiVersion: v1
+keywords:
+  - latest
+sources:
+home:
```

Comment:
also automatically update versions, description and co?

Reply:
maybe re-use the version numbers from AIO?

Reply:
From my experience, many projects version their charts independently from the software itself. This is more confusing for the user but sometimes changes have to be made to the chart alone. Those changes can be reflected in a version bump of the chart.

A pure AIO version bump could then be realized via a minor bump of the chart.